docs/InstallGuide.md

```diff
@@ -75,15 +62,17 @@ Modify the following section in /etc/openvnet/vna.conf on the node.
       addr {
         protocol "tcp"
         host "192.168.100.2"
+        public ""
         port 9103
       }
     }
```

Comment:
Same comments as vnmgr above. Use loopback address and \`\` around the file name.